emsdk.py

```diff
@@ -338,7 +338,13 @@ def win_get_environment_variable(key, system=True):
       # that expansion items such as %PROGRAMFILES% will have been expanded, so
```

Comment:
`winreg.OpenKey` is not "noexcept", so it may throw. The documentation says the type of error would be OSError.
https://docs.python.org/3/library/winreg.html#winreg.OpenKey